cpp/include/cudf/wrappers/bools.hpp

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <utilities/cudf_utils.h>
+
+#include <cub/util_type.cuh>
+
+#include <cstdint>
+#include <iosfwd>
+
+/**---------------------------------------------------------------------------*
+ * @file bools.hpp
```

Comment:
Suggest just naming it bool8.hpp (why the plural "bools"?)

Reply:
I made it plural thinking we’d eventually implement the 1-bit bool wrapper in this file too, but I’m fine with having two different files.

Reply:
I think `bool.hpp` is fine even if there are multiple bool types in it.  (kind of like it's `<cstdint>` not `<cstdints>`.

Reply:
Works for me! File renamed to `bool.hpp`.